index.html

```diff
@@ -2324,10 +2324,10 @@ <h3>Authorization</h3>
         <p>
 <a>Verifiable credentials</a> are intended as a means of reliably identifying
 <a>subjects</a>. While it is recognized that Role Based Access Controls (RBAC) and
-Attribute Based Access Controls (ABAC) rely on this identification as a means
-of authorizing <a>subjects</a> to access resources, this specification
-MUST NOT be used for authorization purposes without an accompanying
-authorization framework.
+Attribute Based Access Controls (ABAC) rely on this identification as a means of
+authorizing <a>subjects</a> to access resources, this specification does not
+provide a complete solution for RBAC or ABAC - authorization is not an appropriate
+use for this specification.
```

Comment:
I think this would be better if it ended with "without an accompanying authorization framework."

Reply:
@brentzundel the latter phrase is essential is my opinion

Reply:
done -- ready to merge?